app/react/src/server/config/webpack.config.js

```diff
@@ -20,6 +20,8 @@ export default function(configDir) {
       preview: [
         require.resolve('./polyfills'),
         require.resolve('./globals'),
+        require.resolve('react-error-overlay'),
```

Comment:
We need to add an explicit dependency on that to `app/react/package.json`

Reply:
done